portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,57 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+
+var TxtType = function(el, toRotate, period) {
```

Comment:
https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments